pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -1123,11 +1188,12 @@ func (n *NetworkPolicyController) getClusterGroupMembers(group *antreatypes.Grou
 	if len(group.IPBlocks) > 0 {
 		return nil, group.IPBlocks
 	} else if len(group.ChildGroups) == 0 {
-		return n.getMemberSetForGroupType(clusterGroupType, group.Name), nil
+		return n.getMemberSetForGroupType(internalGroupType, group.SourceReference.ToGroupName()), nil
 	}
 	var ipBlocks []controlplane.IPBlock
 	groupMemberSet := controlplane.GroupMemberSet{}
 	for _, childName := range group.ChildGroups {
+		childName = k8s.NamespacedName(group.SourceReference.Namespace, childName)
```

Comment:
The method should be renamed to `getInternalGroupMembers` if it can be used for namespaced groups.